openpype/hosts/max/plugins/publish/extract_pointcloud.py

```diff
@@ -181,27 +207,36 @@ def get_files(self,
         return filenames
 
     def partition_output_name(self, container):
-        """
-        Notes:
-            Partition output name set for mapping
-            the published file output
+        """Get partition output name.
+
+        Partition output name set for mapping
+        the published file output.
+
+        Todo:
+            Customizes the setting for the output.
+
+        Args:
+            container: Instance node.
+
+        Returns:
+            str: Partition name.
 
-        todo:
-            Customizes the setting for the output
         """
         partition_count, partition_start = self.get_partition(container)
-        partition = "_part{:03}of{}".format(partition_start,
-                                            partition_count)
-
-        return partition
+        return f"_part{partition_start:03}of{partition_count}"
 
     def get_partition(self, container):
-        """
-        Get Partition Value
+        """Get Partition value.
+
+        Args:
+            container: Instance node.
+
         """
         opt_list = self.get_operators(container)
+        # TODO: This looks strange? Iterating over
+        #   the opt_list but returning from inside?
         for operator in opt_list:
-            count = rt.execute(f'{operator}.PRTPartitionsCount')
-            start = rt.execute(f'{operator}.PRTPartitionsFrom')
+            count = rt.Execute(f'{operator}.PRTPartitionsCount')
+            start = rt.Execute(f'{operator}.PRTPartitionsFrom')
 
             return count, start
```

Comment:
@moonyuet during refactor, I've hit this strange method. Mainly returning inside the for loop seems strange, but maybe there is some reason for it?

Reply:
i did this for just in case there is multiple of export particle nodes within the tyflow node editor. But I think if once the custom modifier introduced, this can be rewritten to be more concise.

Reply:
actually it isn't. it depends whether you want to have more than one export particle nodes